modules/hostap/src/supp_api.h

```diff
@@ -12,6 +12,7 @@
 
 #define MAX_SSID_LEN 32
 #define MAC_ADDR_LEN 6
+#define CHAN_NUM_LEN 6
```

Comment:
`x.abc` + NULL terminator?

Reply:
abcd+" "+NULL terminator

Reply:
May be adding a comment might help including `space is needed to concatenate`

Reply:
Will do

Reply:
Don't see the comment?

Reply:
Added now.